src/OmniSharp.Host/OmniSharp.Host.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>net46</TargetFramework>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <PlatformTarget>AnyCPU</PlatformTarget>
+    <RootNamespace>OmniSharp</RootNamespace>
```

Comment:
`OmniSharp.Host`

Reply:
Today everything is just in the `OmniSharp` namespace, this was just codifying that.  I can change, and update all the namespaces if we want.